user/common-build-problems.md

```diff
@@ -527,3 +527,11 @@ which Docker image you are using on Travis CI.
 1. Manually install dependencies, if any.
 
 1. Manually run your Travis CI build command.
+
+### Running debug VMs
+
+In private repositories and those public repositories for which the feature is enabled,
+it is possible to run a debug VM.
+Using the debug VM, you can interact with the VM.
```

Comment:
This sentence reads funny to me :)

However, I've been trying to think for a alternative and it's not been easy. What about... "Using the debug VM [feature], you can interact with the virtual environment where your build runs."?